app/src/components/FAQ.jsx

```diff
@@ -10,43 +12,17 @@ class FAQ extends Component {
   }
 
   render() {
-    let accordionEntries = [];
-    const faqEntries = this.props.faqEntries;
-    let faqPageContent;
-
-    if (faqEntries) {
-      for (let index = 0; index < faqEntries.length; index++) {
-        accordionEntries.push(
-          <AccordionItem
-            key={index}
-            title={faqEntries[index].questions}
-          >
-            <article >
-              <span
-                dangerouslySetInnerHTML={{
-                  __html: faqEntries[index].answer
-                }}
-              />
-            </article>
-          </AccordionItem>
-        );
-      }
-
-      faqPageContent = (
-        <Accordion allowMultiple>
-          {accordionEntries}
-        </Accordion>
-      );
-    }
-
     return (<div>
       <CoverPrimary
         title="Frequently Asked Questions"
         subtitle="Get answers to commonly asked questions about Global Fishing Watch and commercial fishing."
       />
-      <section>
-        {faqPageContent}
-      </section>
+      {this.props.faqEntries &&
```

Comment:
check the Blog page and use the same "loading" (yet unstyled) if props.faqEntries is empty